### DISCUSSION TITLE
Switch Component Shows Tick/Check When Using Tailwind Forms Plugin

### DISCUSSION THREAD

**Discussion by MattHeslington** [mantinedev/mantine#100] [Feedback]
Hello, 
Love Mantine, will be using it in production shortly.
No big deal, but just to let poeple know, if you're using the Tailwind forms plugin it causes the Switch component to display a check/tick when in positive/on state.

**Comment by rtivital** [comment]
Hi Matt, thanks for feedback, we cannot control external styles that you add to components

**Reply by MattHeslington** [reply]
Of course not, it was just a heads-up for anyone using Mantine with Tailwind

**Reply by Ontopic** [reply]
Yes -  regarding Tailwind and Mantine next to each other - any other quirks or tips you came across?
